manifests/x/xpra-org/xpra/lts/5.1/xpra-org.xpra.lts.installer.yaml

```diff
@@ -0,0 +1,24 @@
+# Created with komac v2.11.2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: xpra-org.xpra.lts
+PackageVersion: '5.1'
+InstallerLocale: en-US
+InstallerType: wix
+Scope: machine
+UpgradeBehavior: install
+ProductCode: '{E6C27B03-2DA0-49EA-A8EB-2380640F451F}'
+ReleaseDate: 2025-04-10
+AppsAndFeaturesEntries:
+- DisplayName: Xpra 5.1 - UNREGISTERED - Wrapped using MSI Wrapper from www.exemsi.com

```

Comment:
Same thing as with the non-LTS version. Even though I definitely prefer msi, in this case the exe installer could be a better choice:

https://xpra.org/lts/windows/Xpra-x86_64_Setup_5.1-r0.exe